src/libraries/System.Transactions.Local/ref/System.Transactions.Local.cs

```diff
@@ -16,7 +16,9 @@ public sealed partial class CommittableTransaction : System.Transactions.Transac
         bool System.IAsyncResult.CompletedSynchronously { get { throw null; } }
         bool System.IAsyncResult.IsCompleted { get { throw null; } }
         public System.IAsyncResult BeginCommit(System.AsyncCallback? asyncCallback, object? asyncState) { throw null; }
+        [System.Runtime.Versioning.UnsupportedOSPlatform("browser")]
```

Comment:
Should the whole class be marked unsupported on browser? A transaction you can’t commit isn’t very useful.

Reply:
Good point